specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/storage.json

```
@@ -0,0 +1,5723 @@
+      "post": {
+        "tags": [
+          "AccountMigrations"
+        ],
+        "operationId": "StorageAccounts_CustomerInitiatedMigration",
+        "description": "Account Migration request can be triggered for a storage account to change its redundancy level. The migration updates the non-zonal redundant storage account to a zonal redundant account or vice-versa in order to have better reliability and availability. Zone-redundant storage (ZRS) replicates your storage account synchronously across three Azure availability zones in the primary region.",
+        "x-ms-examples": {
+          "StorageAccountPostMigration": {
+            "$ref": "./examples/StorageAccountPostMigration.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/MigrationName"
+          },
+          {
+            "name": "validate",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "To specify if it's a 'ValidateMigration' request or an actual migration request; to be used only for cross region migrations currently. It can be either 'true' or 'false'. It will validate the request and return OK or a failure HttpStatusCode based on the validation result."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/StorageAccountMigration"
+            },
+            "description": "The request parameters required to perform storage account migration."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Returned when the request is just for validating the migration request, it does not actually initiate migration; applicable in cross region migration currently."
+          },
+          "202": {
+            "description": "Accepted -- Migration request accepted; operation will complete asynchronously."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+    },
```

A bracket is missing here 
```suggestion
        }
      }
    },
```